docker-compose.yml

```
@@ -116,6 +116,8 @@ services:
       timeout: 5s
       retries: 3
       start_period: 10s
+    volumes:
+      - private_media:/app/private_media
```

I think I'd do the same trick here as with the `web` service and `environment`, so that you can simply do:

```suggestion
    volumes: *web_volumes
```
using yaml references.

that automatically keeps the configurations in sync.